### PR TITLE
SUS-5396 | Migrate cron tab from cron-s1 to k8s cron jobs

### PR DESCRIPTION
**Pull request by JamesHawking**
Example descriptors of cronjobs for maintenance scripts.

https://wikia-inc.atlassian.net/browse/SUS-5396

### CODE REVIEW

**Review by mszabo-wikia — APPROVED** [review]
Looks OK to me. Probably we will have to integrate deployment of these cron jobs into the main prod deploy pipeline once it's ready.

**Comment by sqreek** [comment]
Also adding label sus will be nice

File: docker/maintenance/updateSpecialPages/updateSpecialPages-c1.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+             - name: LOG_SOCKET_ADDRESS
+               value: "tcp://localhost:9999"
+            args:
+            - php
+            - /usr/wikia/slot1/current/src/maintenance/maintenanceTaskScheduler.php
+            - --params="--quiet"
+            - --cluster=c1
+            - --active=2
+            - maintenance/updateSpecialPages.php
+          restartPolicy: Never
```
Comment (macbre) [inline docker/maintenance/updateSpecialPages/updateSpecialPages-c1.yaml]: Just discussed this with @drozdo and we can just run `maintenance/updateSpecialPages.php` across all wikis at once - no need to create seven separate YAML files. Celery queue will make sure we do not run too many scripts at once.

File: docker/maintenance/SMW_conceptCache/SMW_conceptCache-1318.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  schedule: "0 10 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: logger
+            image: artifactory.wikia-inc.com/sus/mediawiki-logger:latest
+          - name: php
+            image: artifactory.wikia-inc.com/sus/mediawiki-sandbox-php:54f12b1.9306b50
```
Comment (macbre) [inline docker/maintenance/SMW_conceptCache/SMW_conceptCache-1318.yaml]: We need to keep this tag in sync with production deploy. Should we make this a template that Jenkinsfile will use and pass a correct tag?

File: docker/maintenance/SMW_conceptCache/SMW_conceptCache-490.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+             - name: SERVER_ID
+               value: "177"
+             - name: LOG_SOCKET_ONLY
+               value: "yes"
+             - name: LOG_SOCKET_ADDRESS
+               value: "tcp://localhost:9999"
+            args:
+            - php
+            - /usr/wikia/slot1/current/src/maintenance/maintenanceTaskScheduler.php
+            - --id=490
```
Comment (macbre) [inline docker/maintenance/SMW_conceptCache/SMW_conceptCache-490.yaml]: We can pass multiple IDs to the `maintenanceTaskScheduler` script. Let's just use a single YAML file for `SMW_conceptCache.php`.

**Pull request merged**